src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -40,7 +47,7 @@ protected override string GetDocumentationForSymbol(string documentationMemberID
                         {
                             if (e.Attribute("name") != null)
                             {
-                                _docComments[e.Attribute("name").Value] = e.Value;
+                                _docComments[e.Attribute("name").Value] = string.Concat(e.Nodes());
```

Comment:
No; VisualStudioDocumentationProvider shares this base class.

http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/DocumentationComments/VisualStudioDocumentationProvider.cs,15